scripts/lintcommit.lua

```diff
@@ -88,7 +88,7 @@ local function validate_commit(commit_message)
   -- Check if type is correct
   local type = vim.split(before_colon, "(", {plain = true})[1]
   local allowed_types = {'build', 'ci', 'docs', 'feat', 'fix', 'perf', 'refactor', 'revert', 'test', 'vim-patch'}
-  if not vim.list_contains(allowed_types, type) then
+  if not vim.tbl_contains(allowed_types, type) then
```

Comment:
What lint failure does `list_contains` cause?

Reply:
It doesn't exist in the version of nvim that the gu actions uses (0.9).